middleware/session/session.go

```diff
@@ -246,6 +250,9 @@ func (s *Session) setSession() {
 }
 
 func (s *Session) delSession() {
+	if s.config == nil {
```

Comment:
When will this be the case?

Reply:
Missing safety check found by https://github.com/uber-go/nilaway